src/screens/AddSiteScreen.js

```diff
@@ -45,4 +58,4 @@ const styles = {
     }
 }
 
-export default Campsite;
\ No newline at end of file
+export default AddSiteScreen;
```

Comment:
Above, the AddSite component was moved and renamed AddSiteScreen. Also, we can see here a pattern used in the rest of the tab screens where we toggle options in the `static navigationOptions` to set things like the icon, header bars, etc.